### DISCUSSION TITLE
Can I make music with Giada without a Daw?

### DISCUSSION THREAD

**Discussion by ghost** [monocasual/giada#494] [General]
Hey,

Is Giada ready for music production, or do I still need a daw?

**Comment by gvnnz** [comment]
Hi @oldcastlehq , Giada is definitely ready for (live) music production! You might want to use a DAW though if you need to record your performance. You can also make the DAW work in sync with Giada through JACK or the MIDI sync protocol, in case you have big audio files to play - for example, an acapella track playing in the DAW and the beat running in Giada.

**Reply by ghost** [reply]
Thanks!
